lib/domain.js

```diff
@@ -446,7 +446,7 @@ Domain.prototype.bind = function(cb) {
 EventEmitter.usingDomains = true;
 
 const eventInit = EventEmitter.init;
-EventEmitter.init = function() {
+EventEmitter.init = function(opts) {
```

Comment:
Could we potentially update to use `...args` to eliminate the chance of this happening again? I realize we don't update the `init` function signature often but in absence of other ways to ensure consistency, I would rather be safe than sorry.

Reply:
As an alternative, perhaps leaving a comment in `lib/events.js` could be another approach.

Reply:
Given that streams API performance is considered critical, I think we should prefer the comment option :+1:

Reply:
nice suggestion, I added the comments in lib/events.js